llvm/lib/Target/PowerPC/PPCAsmPrinter.cpp

```diff
@@ -757,6 +772,17 @@ getTOCEntryTypeForMO(const MachineOperand &MO) {
     llvm_unreachable("Unexpected operand type to get TOC type.");
   }
 }
+
+// FIXME: find alternative approach to get rid of this hack.
+// On AIX, TLS-local-dynamic requires that the symbol for the module handle must
+// have the name "_$TLSML". This symbol is used as one TOC symbol reference
+// itself with an ML relocation type, thus it has "[TC]" attached to its name.
```

Comment:
```suggestion
// FIXME: find an alternative approach to get rid of this hack.
// On AIX, TLS-local-dynamic requires that the symbol for the module handle must
// have the name "_$TLSML". This symbol is used as one TOC symbol reference
// with an @ml relocation specifier, thus it has "[TC]" attached to its name.
```
Maybe this makes a bit more sense?